penny_university_frontend/src/components/Date.tsx

```diff
@@ -6,8 +6,11 @@ type DateProps = {
   className: string,
 }
 
-const Date = ({ date, className }: DateProps) => (
-  <p className={className}><strong>{moment(date).format('M/D/YYYY')}</strong></p>
-)
+const Date = ({ date, className }: DateProps) => {
+  let format = moment(date) > moment() ? 'M/D/YYYY @ h:mm A' : 'M/D/YYYY'
```

Comment:
I mean to remove this component and add the date formatting to the model. We could also the upcoming logic to the model.